client/state/purchases/test/actions.js

```diff
@@ -30,7 +30,9 @@ describe( 'actions', () => {
 		fetchSitePurchases,
 		fetchUserPurchases,
 		removePurchase;
-	useMockery( mockery => {
+	useMockery( function( mockery ) { // must be a function() instead of => because we want the mockery context to set the timeout
+		this.timeout( 10000 );
```

Comment:
We might want to set this timeout in `use-mockery` since the problem seems to be related to mockery itself taking too long, don't you think?